packages/loader/container-loader/src/container.ts

```diff
@@ -172,7 +172,7 @@ export enum ConnectionState {
 export async function waitContainerToCatchUp(container: Container) {
     // Make sure we stop waiting if container is closed.
     if (container.closed) {
-        throw new Error("Container is closed");
+        throw new UsageError("refusingToWaitForClosedContainer");
```

Comment:
Is the use of strings without spacing required by the new API?

Reply:
Or is it desirable for some other reason?

Reply:
I think UsageError and usage of it here is wrong. fluidErrorCode != message to developer, who's locally debugging the issue. fluidErrorCode should be renamed to be callSite, and I think in future may be transitioned to use tags (similar to asserts, to save bundle size).
Message should be pretty clear (to English-speaking developer) on why he/she is getting this error. It has to be in plain English :)